src/components/Disputes/DisputeInfo.js

```diff
@@ -281,9 +305,38 @@ function Field({ label, value }) {
   )
 }
 
+function DisputeActionText({ dispute, isFinalRulingEnsured }) {
+  const theme = useTheme()
+  const { disputedActionText, disputedActionURL } = dispute
+  const lastRound = dispute?.rounds?.[dispute.lastRoundId]
+  const voteWinningOutcome = lastRound?.vote?.winningOutcome
+
+  if (!isFinalRulingEnsured) {
+    return (
+      <Link external href={disputedActionURL}>
+        {disputedActionText}
+      </Link>
+    )
+  }
+
+  const Icon = voteWinningOutcome ? IconCheck : IconCross
+  const prefix = voteWinningOutcome ? 'Alowed action: ' : 'Blocked action: '
+  return (
+    <div
+      css={`
+        display: flex;
+        color: ${voteWinningOutcome ? theme.positive : theme.negative};
+      `}
+    >
+      <Icon />
+      {prefix + disputedActionText}
+    </div>
+  )
+}
```

Comment:
What do you think of using the `DisputeOutcomeText` for this part? Since the logic and the rener structure is basically the same?

What we could do is add a to new props to the component `action` and `verbose`. If verbose is present we render the action (and we set it as `false` by default). 

And in this component we can do :

```
return (
    <DisputeOutcomeText
      action={disputedActionText}
      outcome={appealedRuling || voteWinningOutcome}
      phase={
        appealedRuling ? DisputePhase.AppealRuling : DisputePhase.RevealVote
      }
      verbose
    />
  )
```

Note that we also removed the `appealedRuling` outcome and we should keep this logic because it can happen that the dispute was appealed and nobody confirmed the appeal. In this case the final ruling is what the appealer's outcome

Reply:
good catch